build-system/global-configs/canary-config.json

```diff
@@ -5,6 +5,6 @@
   "amp-live-list": 1,
   "amp-ios-overflow-x": 1,
   "amp-experiment": 1,
-  "expAdsenseA4A": 0.05,
-  "expDoubleclickA4A": 0.05
+  "expAdsenseA4A": 1.0,
```

Comment:
yep this shouldn't be blocking deploy. on bus and will deploy as soon as i
get to desk
On Wed, Sep 7, 2016 at 10:06 AM Terran Lane <notifications@github.com>
wrote:

> In build-system/global-configs/canary-config.json
> <https://github.com/ampproject/amphtml/pull/4837#discussion_r77864335>:
>
> > @@ -5,6 +5,6 @@
> >    "amp-live-list": 1,
> >    "amp-ios-overflow-x": 1,
> >    "amp-experiment": 1,
> > -  "expAdsenseA4A": 0.05,
> > -  "expDoubleclickA4A": 0.05
> > +  "expAdsenseA4A": 1.0,
>
> If you don't have a strong preference, maybe you could go ahead and deploy
> a new release based on this flag change and I'll make the update and submit
> a separate PR to go in for tomorrow's full release?
>
> (I would have already, but I hit some git snag, and I'd rather not delay
> longer, while I untangle it, if possible.)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4837/files/4fff0d3e30cb21b27ba3e7414c645e1b0878339d#r77864335>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ANd2kN7d92wXDAfyL1anec73KEqLHsKvks5qnu7egaJpZM4J2OUn>
> .
>
> --
> You received this message because you are subscribed to the Google Groups
> "amphtml-eng-github" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to amphtml-eng-github+unsubscribe@google.com.
> To post to this group, send email to amphtml-eng-github@google.com.
> To view this discussion on the web visit
> https://groups.google.com/a/google.com/d/msgid/amphtml-eng-github/ampproject/amphtml/pull/4837/r77864335%40github.com
> <https://groups.google.com/a/google.com/d/msgid/amphtml-eng-github/ampproject/amphtml/pull/4837/r77864335%40github.com?utm_medium=email&utm_source=footer>
> .
>
> --
> You received this message because you are subscribed to the Google Groups
> "amphtml-eng" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to amphtml-eng+unsubscribe@google.com.
> To post to this group, send email to amphtml-eng@google.com.
> To view this discussion on the web visit
> https://groups.google.com/a/google.com/d/msgid/amphtml-eng/ampproject/amphtml/pull/4837/r77864335%40github.com
> <https://groups.google.com/a/google.com/d/msgid/amphtml-eng/ampproject/amphtml/pull/4837/r77864335%40github.com?utm_medium=email&utm_source=footer>
> .
>
-- 

Erwin Mombay | UX Engineer | erwinm@google.com | 650-772-9166